istioctl/pkg/install/verify.go

```diff
@@ -250,18 +250,21 @@ func NewVerifyCommand() *cobra.Command {
 	)
 	verifyInstallCmd := &cobra.Command{
 		Use:   "verify-install [-f <deployment or istio operator file>] [--revision <revision>]",
-		Short: "Verifies Istio Installation Status or performs pre-check for the cluster before Istio installation",
+		Short: "Verifies Istio Installation Status",
 		Long: `
 		verify-install verifies Istio installation status against the installation file
 		you specified when you installed Istio. It loops through all the installation
 		resources defined in your installation file and reports whether all of them are
 		in ready status. It will report failure when any of them are not ready.
 
-		If you do not specify installation file it will perform pre-check for your cluster
-		and report whether the cluster is ready for Istio installation.
+		If you do not specify an installation it will check for an IstioOperator resource
+		and will verify if pods and services defined in it are present.
+
+		Note: For verifying whether your cluster is ready for Istio installation, see
+		istioctl experimental precheck.
 `,
 		Example: `
-		# Verify that Istio can be freshly installed
+		# Verify that Istio is installed correctly via Istio Operator
```

Comment:
Is this still correct without passing any param?
I'm getting this with latest master code base.
```
$ go run "${GOPATH}"/src/istio.io/istio/istioctl/cmd/istioctl/main.go verify-install                                                     

Error: could not load IstioOperator from cluster: control plane revision "" not found.  Use --filename
exit status 1
```

Reply:
Yes, I'm getting the same. That's what motivated this PR. The description is outdated.
See https://github.com/istio/istio/issues/24068#issuecomment-634015870 and https://github.com/istio/istio/pull/23087#issue-406266978

Reply:
Ok, I see.  Can we clarify the msg?  Via Istio operator, does it mean server side operator or istioctl?  Do we need to specify istio operator here?